frontend/src/types/LogRecordTypes.ts

```diff
@@ -1,13 +1,19 @@
-export type LogRecord = {
-  logId: number;
+interface AttnToDetails {
   attnTo?: number;
   attnToFirstName?: string;
   attnToLastName?: string;
-  building: string;
-  datetime: string;
+};
+
+interface EmployeeDetails {
   employeeId: number;
   employeeFirstName: string;
   employeeLastName: string;
+};
+
+export interface LogRecord extends AttnToDetails, EmployeeDetails {
```

Comment:
So instead of extending this, it's better practice to actually just define it as the property type. It would look like this:
```
attnTo: AttnToDetails
employee: EmployeeDetails
```

Then, in the separate interfaces themselves, you can define them like:
```
interface EmployeeDetails {
  id: number;
  firstName: string;
  lastName: string;
}
```

and the same would go for `AttnToDetails`

Reply:
Would that make it into a nested object though?